### ISSUE TITLE
New release on hex.pm?

### ISSUE THREAD

**Issue by aseigo** [ateliware/triplex#86]
The last release on hex.pm is from May 31, 2019. There have been a few commits since then, and there are also a couple of PRs pending. Is there anything the users of the lib could do to help move this along? 

Cheers!

**Comment by aseigo** [comment]
ping @kelvinst ?

Every time I see the compile errors that I know are fixed in the repo it makes me a little sad :) Hope to see a release soon. Is there anything we can do to help out?

**Comment by kelvinst** [comment]
Hey, so sorry! I'm really busy since mid 2021, changed to another company. I can't give you a date, but I promise I will try out to organize what's already done and do a release soon.

**Comment by aseigo** [comment]
Sorry to bug you some more ... but this continues to stretch out. Is there something I can do to help out? Should I fork the repository? I'd prefer to help out somehow rather than fork, but after 3 years with no releases I feel like it's getting to the point that something needs to be done :/

**Comment by kelvinst** [comment]
@petersonfs can someone from ateliware check it? It's being a while we don't release a version of triplex and it would be good to have the pending PRs merged and released.

**Comment by aseigo** [comment]
Bumping on this one again. I have an approved PR coming up on four years now, with the last release on hex.pm 6 years ago.

Is this library abandoned? If it is, then perhaps the maintainer(s) can put out a request in the community (e.g. on the forums) to take up maintenance of this library?

**Comment by sergchernata** [comment]
Happy to volunteer here.

**Comment by aseigo** [comment]
@sergchernata I feel that at this point the library is abandoned and needs to be forked to survive. Unfortunate, but it happens. I'd be happy to collaborate and support (testing, PRs, etc.) a fork that would resurrect this lib.

**Comment by sergchernata** [comment]
@aseigo I'm down to do that, the only bummer is that we won't be able to take over the hex.pm listing.

**Comment by aseigo** [comment]
@sergchernata I have merged all open PRs into my fork here https://github.com/aseigo/triplex

I also fixed the tests and made it possible to run the tests against the backends you desire (e.g. I do not have MySQL installed on this particular machine but should still be able to run the test suite).

Now to decide how to move forward.

I'd be happy to add @sergchernata as a colaborator on my fork of the repo (because bus numbers of one are horrible), and then it's a question of how to go about a new release.

@kelvinst One way forward would be to share/pass on the hex.pm package ownership. Another would be to pull in the changes from my fork into yours, actually make a release, and then consider adding some of us who are able and willing to put int he work as collaborators so we do not have to wait on you for everything. 

I see that you are active on github, and kindly request that you do not simply ignore this situation.

If I do not hear back within the the next week, I will release my fork on hex.pm under a new name. That is probably the worst possible result, but we need to move on here.

**Comment by aseigo** [comment]
As a small addendum, I've also committed fixes (including one from a PR the contributor closed due to it being ignored) for a couple more issues in my repository. So my fork now goes a bit further than just the open PRs + fixed tests.
